menpo/base.py

```diff
@@ -247,7 +241,6 @@ def _verify_target(self, new_target):
                 "- new target has to have the same number of points as the"
                 " old".format(self.target.n_points, new_target.n_points))
 
-    @abc.abstractmethod
```

Comment:
@patricksnape maybe we should `raise NotImplementedError()` in all these abstract method cases to still make the contract we are expecting clear?